1.10/deploying-services/creating-services/health-checks.md

```diff
@@ -1,23 +1,23 @@
 ---
-post_title: Marathon Health Checks
+post_title: Health Checks
 menu_order: 1
 ---
 
 You can define health checks for your DC/OS services. Health checks are defined on a per application basis, and are run against that application's tasks.
 
-Marathon health checks perform periodic checks on the containers distributed across a cluster to make sure they’re up and responding. If a container is down for any reason, Marathon will detect this and stop sending traffic to the container and try to restart it.
+Health checks perform periodic checks on the containers distributed across a cluster to make sure they’re up and responding. If a container is down for any reason, Marathon will detect this and stop sending traffic to the container and try to restart it.
```

Comment:
This doesn't sound quite right:

> Marathon will detect this and stop sending traffic to the container and try to restart it.

Marathon-lb or minuteman is responsible for sending traffic; specifically, Marathon does nothing here.

I might say:

> If health checks for any reason, Mesos will report the task as unhealthy so that status-aware load-balancers can stop sending traffic to the container. Additionally, after the configured maximum consecutive failures, Marathon will kill the task and restart it.